ofonotest/main.cpp

```diff
@@ -8,5 +9,9 @@ Q_DECL_EXPORT int main(int argc, char *argv[])
     QQmlApplicationEngine engine;
     engine.load(QUrl(QLatin1String("/opt/examples/libqofono-qt5/qml/ofonotest/main.qml")));
 
+    if ( QWindow* window = qobject_cast<QWindow*>( engine.rootObjects().at(0) ) ) {
```

Comment:
Hm, this appears to evaluate false here. rootObjects().at(0) is QQuickRectangle

Reply:
I have changed the main.qml to contain Window. In my tests the binary already shows the window, so it works for me.

Reply:
I am not sure, but the Window object is maybe enough it self without changes in C++ part. I can retest it, but not during working hours

Reply:
It must be something like:
```
Window {
   width: 320
   height: 200
   visible: true
}
```
My recent `push --force` contain required changes